src/app/project/[id]/edit/page.tsx

```
@@ -167,16 +160,83 @@ export default function ProductEdit() {
+        description: state.description,
+        goalAmount: Number.parseInt(state.targetAmount.replace(/,/g, ""), 10),
+        contentMarkdown: state.markdown,
+        startAt: state.startDate.toISOString(),
+        endAt: state.endDate.toISOString(),
+        shippedAt: state.deliveryDate.toISOString(),
+        images: imageData, // URL과 File 객체 모두 포함
+        products: state.supportOptions.map((option) => ({
+          name: option.name,
+          description: option.description,
+          price: Number.parseInt(option.price.replace(/,/g, ""), 10),
+          stock: Number.parseInt(option.stock.replace(/,/g, ""), 10),
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)